server/src/repositories/album.repository.ts

```diff
@@ -302,4 +302,47 @@ export class AlbumRepository implements IAlbumRepository {
 
     return result.affected;
   }
+
+  @GenerateSql({ params: [DummyValue.UUID, DummyValue.STRING, undefined] })
+  getByName(userId: string, albumName: string, shared?: boolean): Promise<AlbumEntity[]> {
```

Comment:
This should be paginated, no? Only returning the first 1k results is optimistic 😅

Reply:
It would be nice but I don't think that really matters: the goal of this endpoint is not to get all albums.

Reply:
IMO we should try to have all of our endpoints behave in a similar manner. Search person vs search album vs search something else should all have the same method signatures.

Reply:
I strongly agree and I there are people out there with >10k albums, for whom a search result may very well include >1k albums. I think it's bad just not supporting that.

Reply:
> I think it's bad just not supporting that.

I agree. But that's for a future PR :stuck_out_tongue_closed_eyes:

Reply:
Generally speaking it would be nice to paginate ALL endpoints that return a list of items

Reply:
> I agree. But that's for a future PR 😝

Why? :P

Reply:
> Why? :P

Because I agree with @jrasm91 too. Do we want the same method signatures or not ?

Reply:
Hm that's fair.

Reply:
Well... I added pagination for both endpoints.